gcloud-java-dns/src/main/java/com/google/gcloud/spi/DnsRpc.java

```diff
@@ -16,6 +16,12 @@
 
 package com.google.gcloud.spi;
 
+import com.google.api.services.dns.model.Change;
+import com.google.api.services.dns.model.ManagedZone;
+import com.google.api.services.dns.model.Project;
+import com.google.api.services.dns.model.ResourceRecordSet;
+import com.google.gcloud.dns.DnsException;
+
```

Comment:
I think the following RPC string value need to change:
"maxSize" -> "maxResults"
"dnsName" -> "name"